testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestCase.java

```diff
@@ -268,7 +269,7 @@ private String getClientJarPaths() throws IOException {
         jar.addManifest()
                 .addAsManifestResource(WSTrustTestCase.class.getPackage(), "META-INF/clientKeystore.properties", "clientKeystore.properties")
                 .addAsManifestResource(WSTrustTestCase.class.getPackage(), "META-INF/clientstore.jks", "clientstore.jks");
-        File jarFile = new File("jaxws-samples-wsse-policy-trust-client.jar");
+        File jarFile = new File(TestSuiteEnvironment.getTmpDir(), "jaxws-samples-wsse-policy-trust-client.jar");
```

Comment:
Same here.

Reply:
it's used to create an URLClassLoader with these security files as resource and put into setContextClassLoader by TestWatcher, then later it retrieves back from [WSTrustTestUtils static methods](https://github.com/wildfly/wildfly/blob/6b61a6003f704221f66dcd9f418bcb7af88fb9ab/testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestUtils.java#L264) for client env setup.